<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepairMaterialDialog</class>
 <widget class="QDialog" name="RepairMaterialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add / Edit Repair Material</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>155</y>
     <width>166</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cost_sbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>5000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>126</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>126</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Material</string>
   </property>
  </widget>
  <widget class="QComboBox" name="material_cbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="amount_sbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>126</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Cost [TZS]</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>amount_sbox</tabstop>
  <tabstop>material_cbox</tabstop>
  <tabstop>cost_sbox</tabstop>
 </tabstops>
 <resources>
  <include location="./resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RepairMaterialDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RepairMaterialDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
